### ISSUE TITLE
snippetPreview + SEO accessibility

### ISSUE THREAD

**Issue by terw-dan** [Yoast/YoastSEO.js#103]
There are a couple of things we can use to improve the accessibilty of the snippetpreview
- use inputs in stead of contenteditable divs
- add placeholders to inputs
- add labels
- add aria feedback for maximum number of chars

SEO
- add screenreader class to overall SEO. 
- add notice per subscore after something has changed (currently it is only recalculated)
- add polite aria-live to the overall SEO score.

**Comment by atimmer** [comment]
@afercia Can you take a look at this and see if these are good ideas and which ones we have already done correctly? I think it might be better to create single issues for everything on this list that hasn't been fixed yet.

**Comment by afercia** [comment]
Totally agree about creating single issues. The content analysis and scores are the hardest thing in all the metabox. There's a huge amount of information that gets updated, thinking we should make a well thought decision before starting coding. Will talk about this with @omarreiss and @tacoverdo soon :)

**Comment by atimmer** [comment]
@afercia What is the status on this?

**Comment by afercia** [comment]
@atimmer I think the inputs/labels/screen-reader-text part is pretty good now. The aria-live part is still to implement but I guess it would require some discussion and decision about how it should work. We can't send an audible message for each change in the analysis, that would result in  bombarding users with a huge amount of messages.

**Comment by atimmer** [comment]
I think we can make single issues about this if there is stuff to fix. Closing this issue for now.
